trade-finance-manager-api/src/v1/controllers/deal.submit.controller.js

```diff
@@ -101,6 +101,16 @@ const submitDealAfterUkefIds = async (dealId, dealType, checker) => {
 
   if (dealHasBeenResubmit) {
     const { tfm: tfmDeal } = await findOneTfmDeal(dealId);
+
+    /**
+     * checks if can update to MIN
+     * if it can, changes mappedDeal to show MIN to allow gef fee record to be calculated
+     * canBeMIN then also used to update deal to MIN
+    */
+    const canBeMIN = shouldUpdateDealFromMIAtoMIN(mappedDeal, tfmDeal);
+    if (canBeMIN) {
```

Comment:
can we rename this to `isUpdatingToMIN` so it's a bit clearer

"can be" to me implies that it can be a MIN it's not 100% clear that it _will_ be changing

Reply:
done